projects/media-viewer/src/lib/shared/toolbar.events.service.ts

```diff
@@ -0,0 +1,36 @@
+import {Injectable} from '@angular/core';
+import {BehaviorSubject} from 'rxjs';
+
+// Toolbar Custom-Event Types
+export type HighlightMode = boolean;
+export type DrawMode = boolean;
+
+@Injectable({providedIn: 'root'})
+export class ToolbarEventsService {
```

Comment:
I'm not 100% sure on the angular naming convention but I think this file should be called `toolbar-events.service.ts` - it looks a bit weird but I think it's convention.

Reply:
i agree, it probably should be a -. I'll revise.

Reply:
renamed. @linusnorton can you recheck?